### PR TITLE
myjwt #2942

### PR DESCRIPTION
**Pull request by dualfade**
please review.

### CODE REVIEW

File: packages/myjwt/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+_pkgname=MyJWT
+pkgver=119.f26dd19
+pkgrel=1
+pkgdesc='This cli is for pentesters, CTF players, or dev. You can modify your jwt, sign, inject, etc...'
+arch=('any')
+groups=('blackarch' 'blackarch-exploitation' 'blackarch-cracker')
+url='https://github.com/mBouamama/MyJWT'
+license=('MIT')
+depends=('python' 'python-click' 'python-cryptography' 'python-pyopenssl'
+  'python-requests')
```
Comment (Edu4rdSHL) [inline packages/myjwt/PKGBUILD:14]: Please keep the indentation.

File: packages/myjwt/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=myjwt
+_pkgname=MyJWT
+pkgver=119.f26dd19
+pkgrel=1
+pkgdesc='This cli is for pentesters, CTF players, or dev. You can modify your jwt, sign, inject, etc...'
```
Comment (Edu4rdSHL) [inline packages/myjwt/PKGBUILD:8]: Use only one dot at the end of the phrase.

**Review by Edu4rdSHL — CHANGES_REQUESTED** [review]

File: packages/myjwt/PKGBUILD (hunk 2 above)
Comment (dualfade) [reply packages/myjwt/PKGBUILD:8]: Fixed.

Comment (Edu4rdSHL) [reply packages/myjwt/PKGBUILD:8]: Do you forget to commit?

File: packages/myjwt/PKGBUILD (hunk 1 above)
Comment (dualfade) [reply packages/myjwt/PKGBUILD:14]: leaving indentation.

File: packages/myjwt/PKGBUILD (hunk 2 above)
Comment (dualfade) [reply packages/myjwt/PKGBUILD]: No commented before commit; sorry about the confusion there ;)

**Comment by Edu4rdSHL** [comment]
Thank you a lot @dualfade <3

**Pull request merged**